guides/doc-Project_API/master.adoc

```diff
@@ -0,0 +1,22 @@
+include::common/attributes.adoc[]
+include::common/header.adoc[]
+:context: rest-api
+:rest-api:
+
+= {APIDocTitle}
+
+include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[leveloffset=+1]
+
+include::topics/assembly_introduction-to-project-api.adoc[leveloffset=+1]
```

Comment:
OK for now. In a follow-up PR, I can move everything to "guides/common/modules".

Reply:
I will do that before I merge it.

Reply:
Done.